src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -39,6 +27,31 @@ $pf-c-dropdown--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "
   --pf-c-dropdown__toggle--m-primary--focus--BackgroundColor: var(--pf-global--primary-color--200);
   --pf-c-dropdown--m-expanded__toggle--m-primary--BackgroundColor: var(--pf-global--primary-color--200);
 
+  // ::before
+  --pf-c-dropdown__toggle--before--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-dropdown__toggle--before--BorderTopColor: var(--pf-global--BorderColor--300); // remove at breaking change
+  --pf-c-dropdown__toggle--before--BorderRightColor: var(--pf-global--BorderColor--300); // remove at breaking change
+  --pf-c-dropdown__toggle--before--BorderBottomColor: var(--pf-global--BorderColor--200); // remove at breaking change
+  --pf-c-dropdown__toggle--before--temp--BorderBottomColor: var(--pf-global--BorderColor--300); // change to --pf-c-dropdown__toggle--before--BorderColor at breaking change
+  --pf-c-dropdown__toggle--before--BorderLeftColor: var(--pf-global--BorderColor--300); // remove at breaking change
+  --pf-c-dropdown__toggle--hover--before--BorderBottomColor: var(--pf-global--active-color--100); // remove at breaking change
+  --pf-c-dropdown__toggle--active--before--BorderBottomWidth: var(--pf-global--BorderWidth--md); // remove at breaking change
+  --pf-c-dropdown__toggle--active--before--BorderBottomColor: var(--pf-global--active-color--100); // remove at breaking change
+  --pf-c-dropdown__toggle--focus--before--BorderBottomWidth: var(--pf-global--BorderWidth--md); // remove at breaking change
+  --pf-c-dropdown__toggle--focus--before--BorderBottomColor: var(--pf-global--active-color--100); // remove at breaking change
+  --pf-c-dropdown--m-expanded__toggle--before--BorderBottomWidth: var(--pf-global--BorderWidth--md); // remove at breaking change
+  --pf-c-dropdown--m-expanded__toggle--before--BorderBottomColor: var(--pf-global--active-color--100); // remove at breaking change
+
+  // ::after
+  --pf-c-dropdown__toggle--after--BorderBottomColor: var(--pf-c-dropdown__toggle--before--BorderBottomColor); // update value at breaking change
+  --pf-c-dropdown__toggle--hover--after--BorderBottomColor: var(--pf-c-dropdown__toggle--hover--before--BorderBottomColor); // update value at breaking change
+  --pf-c-dropdown__toggle--active--after--BorderBottomWidth: var(--pf-c-dropdown__toggle--active--before--BorderBottomWidth); // update value at breaking change
+  --pf-c-dropdown__toggle--active--after--BorderBottomColor: var(--pf-c-dropdown__toggle--active--before--BorderBottomColor); // update value at breaking change
+  --pf-c-dropdown__toggle--focus--after--BorderBottomWidth: var(--pf-global--BorderWidth--md); // update value at breaking change
```

Comment:
Should the value for this be `--pf-c-dropdown__toggle--focus--before--BorderBottomWidth`?